asab/application.py

```diff
@@ -194,6 +199,28 @@ def handler(type):
 		# Every 10 minutes listen for housekeeping
 		self.PubSub.subscribe("Application.tick/600!", self._on_housekeeping_tick)
 
+		# Run the watchdog to detect lost interactivity on the event loop
+		thread = threading.Thread(target=self._run_watchdog_for_event_loop)
+		thread.daemon = True  # Daemonize the thread to ensure it exits with the main program
+		thread.start()
+
+
+	def _run_watchdog_for_event_loop(self):
```

Comment:
`def _watchdog(self):`

... don't try to squeeze documentation into the name of the method, please.